testing/load-aware/test.py

```diff
@@ -62,15 +62,23 @@ def prepare_ci():
     Prepares the cluster and the namespace for running Load-Aware scale tests
     """
 
+    test_namespace = config.ci_artifacts.get_config("load_aware.scale_test.namespace")
+    run.run(f"oc create ns {test_namespace}")
+
+    run.run("./run_toolbox.py from_config cluster set_scale --prefix='sutest'")
+    run.run("./run_toolbox.py from_config cluster set_project_annotation --prefix sutest --suffix node_selector")
+    run.run("./run_toolbox.py from_config cluster set_project_annotation --prefix sutest --suffix toleration")
+
     install_ocp_pipelines()
 
     run.run("./run_toolbox.py from_config cluster capture_environment --suffix sample")
     run.run("./run_toolbox.py from_config load_aware deploy_trimaran")
 
-    test_namespace = config.ci_artifacts.get_config("load_aware.scale_test.namespace")
-    run.run(f"oc create ns {test_namespace}")
     run.run("./run_toolbox.py from_config utils build_push_image --suffix deps")
     run.run("./run_toolbox.py from_config utils build_push_image --suffix make")
+    run.run("./run_toolbox.py from_config cluster preload_image --suffix deps")
+    run.run("./run_toolbox.py from_config cluster preload_image --suffix make")
+    run.run("./run_toolbox.py from_config cluster preload_image --suffix sleep") # The sleep image is just fedora:latest
```

Comment:
@drewrip , here is reference to `fedora:latest` that should be removed ;-)

Reply:
Gotcha!